docs/book/src/topics/multitenancy.md

```diff
@@ -6,6 +6,8 @@ For details, see the [multi-tenancy proposal](https://github.com/kubernetes-sigs
 
 For multi-tenancy support, a reference field (`identityRef`) is added to `AWSCluster`, which describes the identity to be used when reconciling the cluster.
 
+In other words, `identityRef`is the function that dictates _which_ aws account cluster-api will deploy into.
```

Comment:
I believe this might be redundant, the `identityRef` has nothing to do with the account per se, and the paragraph at line 24 clarify what is this about.

Reply:
I understand the point you're making.  I think while it may be redundant, it states an important part of the multi-tenancy configuration.  Which is how an operator dictates which cluster they can deploy into.

I'd like to leave something high level explaining this at the top of the doc, do you have any suggestions for wording?

Reply:
You could say something like:

> `identityRef` reference the credentials or the role that the capa-controller will use to manage the relative cluster

But then we are, sort of, repeating what's in the previous paragraph